src/gui/dialogs/preferences_dialog.cpp

```diff
@@ -761,8 +761,8 @@ listbox& preferences_dialog::setup_hotkey_list()
 	t_string& row_is_g_markup = row_data["lbl_is_game"]["use_markup"];
 	t_string& row_is_e        = row_data["lbl_is_editor"]["label"];
 	t_string& row_is_e_markup = row_data["lbl_is_editor"]["use_markup"];
-	t_string& row_is_t        = row_data["lbl_is_titlescreen"]["label"];
-	t_string& row_is_t_markup = row_data["lbl_is_titlescreen"]["use_markup"];
+	t_string& row_is_t        = row_data["lbl_is_mainmenu"]["label"];
+	t_string& row_is_t_markup = row_data["lbl_is_mainmenu"]["use_markup"];
```

Comment:
There's a `t` in the variable names.

I'm not sure if the `lbl_is_titlescreen` needs to change - it already isn't a user-facing string.

Reply:
Thanks for catching that - I didn't include `*.cfg` in my text search.

Even if it's not user-visible, source code should be consistent. Inconsistency will leave open the possibility for confusion later on, otherwise.

Should be updated in a few minutes.